src/components/AggregationPanel/AggregationPanelComponents.js

```diff
@@ -95,3 +97,116 @@ export const ButtonElement = ({ item, showNote }) => {
     </div>
   );
 };
+
+export const PanelElement = ({ item, showNote, objectId, depth = 0 }) => {
```

Comment:
Instead of introducing another `PanelElement` I would suggest to extend existing one (in `AggregationPanel.js`), and adopt it to loading data subset and being used recessively. This way we should avoid code duplication.

Reply:
I have tried doing in the latest commits, can you please review it?